feature-libs/pickup-in-store/occ/adapters/default-occ-pickup-location-config.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { OccConfig } from '@spartacus/core';
+
+/**
+ * The endpoints to call from the OCC adapter for pickup locations.
+ */
+export const defaultOccPickupLocationConfig: OccConfig = {
+  backend: {
+    occ: {
+      endpoints: {
+        storeDetails: 'stores/${storeName}',
+        updateDeliveryOption:
```

Comment:
`updateDeliveryOption` can be removed, since we have `updateEntries` in `defaultOccCartConfig`.

Reply:
Done